### DISCUSSION TITLE
Multi-tenancy bundle with softDeletable

### DISCUSSION THREAD

**Discussion by spoltix** [RamyHakam/multi_tenancy_bundle#52] [Q&A]
Hello,

I'm trying to use [StofDoctrineExtension / Deletable](https://symfony.com/bundles/StofDoctrineExtensionsBundle/current/softdeleteable-filter.html) with the multi-tenancy bundle, but it seems that it isn't working.

Have you tested this?

**Comment by RamyHakam** [comment]
Hey 
I did not test it,
But I think we can add  it like DQL function support which we added here at #29 

You can check this PR and if you have time you can use it as an example to push a PR to support [StofDoctrineExtension / Deletable](https://symfony.com/bundles/StofDoctrineExtensionsBundle/current/softdeleteable-filter.html)

**Reply by spoltix** [reply]
Yes, you first need to modify DependencyInjection/Configuration.php so that the filters are supported.

As soon as I have time, I will try to submit a PR.
